.github/workflows/webonary-cloud-api-pr.yml

```diff
@@ -0,0 +1,96 @@
+name: Webonary Cloud API PR
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - master
+    paths:
+      - "webonary-cloud-api/**"
+  push:
+    branches:
+      - develop
+      - master
+    paths:
+      - "webonary-cloud-api/**"
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set env to stage if PR is for merging to develop
```

Comment:
This says set env to stage. Did you mean "dev"? line 29 says `DEPLOY_ENV=dev`

Reply:
nice catch. corrected